ReactWindows/ReactNative/Modules/StatusBar/StatusBarModule.cs

```diff
@@ -0,0 +1,167 @@
+﻿using ReactNative.Bridge;
+using ReactNative.UIManager;
+using System;
+using Windows.Foundation.Metadata;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using Windows.UI;
+using Windows.UI.Core;
+
+namespace ReactNative.Modules.StatusBar
+{
+    /// <summary>
+    /// A module that allows JS to set statusbar properties.
+    /// </summary>
+    class StatusBarModule : NativeModuleBase
+    {
+        private bool _hidden;
+        private bool _translucent;
+        private Color? _color;
+
+        /// <summary>
+        /// Instantiates the <see cref="StatusBarModule"/>.
+        /// </summary>
+        internal StatusBarModule()
+        {
+            //Mobile customization
+            if (ApiInformation.IsTypePresent("Windows.UI.ViewManagement.StatusBar"))
+            {
+                var statusBar = Windows.UI.ViewManagement.StatusBar.GetForCurrentView();
+                if (statusBar != null)
+                {
+                    _color = statusBar.BackgroundColor;
+                    _translucent = statusBar.BackgroundOpacity == 1 ? false : true;
+                }
+            }
+
+            //PC customization
+            else if (!DeviceInfo.IsRunningOnEmulator && ApiInformation.IsTypePresent("Windows.UI.ViewManagement.ApplicationView"))
+            {
+                var titleBar = Windows.UI.ViewManagement.ApplicationView.GetForCurrentView().TitleBar;
+                if (titleBar != null)
+                {
+                    _color = titleBar.BackgroundColor;
+                }
+            }
+        }
+
+        /// <summary>
+        /// The name of the native module.
+        /// </summary>
+        public override string Name
+        {
+            get
+            {
+                return "StatusBarManager";
+            }
+        }
+
+        /// <summary>
+        /// Hide or show statusbar.

```

Comment:
I think it should be StatusBarManager. Otherwise we need to make Windows specific changes to `StatusBar.js`

Line 20   `const StatusBarManager = require('NativeModules').StatusBarManager;`